libs/components/core/src/lib/modules/log/types/log-level-token.ts

```diff
@@ -0,0 +1,5 @@
+import { InjectionToken } from '@angular/core';
+
+import { SkyLogLevel } from './log-level';
+
+export const SKY_LOG_LEVEL = new InjectionToken<SkyLogLevel>('SkyLogLevel');
```

Comment:
Should be marked `@internal` so it doesn't get exposed to TypeDoc.

Reply:
Done